### PR TITLE
Add scheduled database token cleanup job

### PR DESCRIPTION
**Pull request by Yamsha75**
- uses apscheduler and flask_apscheduler libraries
- deletes expired tokens at app launch and then every 12 hours

### CODE REVIEW

File: api/everycache_api/auth/token_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    with apscheduler.app.app_context():
+        tokens = Token.query.filter(Token.expires < datetime.utcnow()).all()
+
+        if tokens:
+            current_app.logger.info(
+                f"Found {len(tokens)} expired tokens in database. Cleaning up..."
+            )
+
+            for token in tokens:
+                db.session.delete(token)
```
Comment (Nieminik) [inline api/everycache_api/auth/token_cleanup.py:21]: wouldn't `tokens.delete()` work?

File: api/everycache_api/auth/token_cleanup.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            db.session.commit()
+
+            current_app.logger.info("Done")
+        else:
+            current_app.logger.info("No expired tokens found in database.")
+
+
+def add_token_cleanup_job():
+    # add job to scheduler
+    trigger = IntervalTrigger(hour="*/12", timezone=utc)
```
Comment (Nieminik) [inline api/everycache_api/auth/token_cleanup.py:32]: https://apscheduler.readthedocs.io/en/latest/modules/triggers/interval.html
`hours (int) – number of hours to wait`
Could you explain the "*/12" value? ; p

File: api/everycache_api/auth/token_cleanup.py (hunk 1 above)
Comment (Yamsha75) [reply api/everycache_api/auth/token_cleanup.py]: No, but I changed it to be more efficient anyway, see 495ec88

File: api/everycache_api/auth/token_cleanup.py (hunk 2 above)
Comment (Yamsha75) [reply api/everycache_api/auth/token_cleanup.py]: It was a mistake, didn't change that argument when switching from cron trigger. Fixed in 495ec88

**Review by Nieminik — APPROVED** [review]

**Pull request merged**